pipelines/appco/Dockerfile.setup

```diff
@@ -0,0 +1,39 @@
+FROM golang:1.23-alpine3.19
+
+ARG KUBECTL_VERSION=v1.30.0
+
+ARG RKE_VERSION=v1.3.4
+
+ARG TERRAFORM_VERSION=1.3.5
+
+ARG YQ_VERSION=v4.24.2
+
+ENV WORKSPACE /src/longhorn-tests
+
+WORKDIR $WORKSPACE
+
+RUN mkdir -p /usr/local/share/ca-certificates/suse
+# SUSE_Trust_Root.crt stored on jenkins
+COPY SUSE_Trust_Root.crt /usr/local/share/ca-certificates/suse/SUSE_Trust_Root.crt
```

Comment:
Is it required to copy the crt file to `/usr/local/share/ca-certificates/suse/` folder in the docker container? What if it's copied to a random folder?

Reply:
Apart from copying the crt file to VMs, when is the crt file used in the docker container?

Reply:
Hi @yang, in the begining, the install method is `helm install longhorn oci://registry.suse.de/home/mbjoerkelund/orchid/charts/charts/longhorn:1.8.0 -n longhorn-system`, hence it need the crt file in container to pull the packaged chart, however the current design is replace images in longhorn.yaml, therefore:
- crt file is only for copy to vms in the container
- any path sould be OK since no need to do `update-ca-certificates`, I can update with the path suggestion

Reply:
It's okay to keep it in `/usr/local/share/ca-certificates/` folder, but should we call `update-ca-certificates` explicitly somewhere in the container to make it take effect?

Reply:
Running `update-ca-certificates` in the container is not required as it only pulls Longhorn, which does not need the certificate. However, since the worker node does require it, I have added `update-ca-certificates` in `test_framework/terraform/harvester/sles/main.tf` under the `e2e-machine-config-worker` section for review, thank you.